src/log.c

```diff
@@ -34,27 +34,31 @@
 #include <wolfssh/log.h>
 #include <wolfssh/error.h>
 
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdarg.h>
+#include <time.h>
```

Comment:
This now always tries to include time.h and other headders. Should it be conditional for portability/embedded development?

Reply:
The time stuff is optional, so I wrapped it in a guard check.